librz/util/time.c

```diff
@@ -209,6 +209,15 @@ RZ_API struct tm *rz_localtime_r(const time_t *time, struct tm *res) {
 #endif
 }
 
+RZ_API struct tm *rz_gmtime_r(const time_t *time, struct tm *res) {
+#if __WINDOWS__
+	errno_t err = gmtime_s(res, time);
+	return err ? NULL : res;
+#else
+	return gmtime_r(time, res);
```

Comment:
this should probably go under `HAVE_GMTIME_R` because I don't think every system has this.

Reply:
Not a problem. What do you suggest to use if there is no `gmtime_r()`?

Reply:
That's a nice question :D Just gmtime and hope things go well :PP I don't think we have real alternatives!

Reply:
Do you have any example of such a system? The `gmtime_r()` works on most of the stuff I saw so far. It's an ancient function. I suggest we don't do that until there is actual report of it missing. It was used in the code all these years and nobody complained so far.

Reply:
Sorry...I mis-read the man page!! Sure, ignore my comment!